compiler/qsc_parse/src/item.rs

```diff
@@ -98,10 +98,20 @@ pub(super) fn parse_fragments(s: &mut Scanner) -> Result<Vec<Fragment>> {
 }
 
 fn parse_fragment(s: &mut Scanner) -> Result<Fragment> {
-    if let Some(namespace) = opt(s, parse_namespace)? {
+    let doc = parse_doc(s);
```

Comment:
Maybe a short comment explaining why this is necessary and that we hope it can be cleaned up in the future? (since the fact that this will hopefully be obsoleted by modules-as-items made us more OK with a hacky solution for now)

Reply:
Added.